Source/carthage/Bootstrap.swift

```diff
@@ -17,7 +17,9 @@ public struct BootstrapCommand: CommandProtocol {
 				if !FileManager.default.fileExists(atPath: project.resolvedCartfileURL.path) {
 					let formatting = options.checkoutOptions.colorOptions.formatting
 					carthage.println(formatting.bullets + "No Cartfile.resolved found, updating dependencies")
-					return project.updateDependencies(shouldCheckout: options.checkoutAfterUpdate, buildOptions: options.buildOptions)
+					return project.updateDependencies(shouldCheckout: options.checkoutAfterUpdate,
+													  useNewResolver: options.useNewResolver,
```

Comment:
No idea why it looks like this on Github, looks fine in Xcode.

Reply:
I think this should be formatted like this:

```swift
return project.updateDependencies(
	shouldCheckout: options.checkoutAfterUpdate,
	useNewResolver: options.useNewResolver,
	buildOptions: options.buildOptions
)
```